setup.py

```diff
@@ -41,7 +41,7 @@
         ],
     },
     install_requires=[
-        "flask~=2.2",
+        "flask>=2.2",
         "jsonschema>3",
         "sqlalchemy<1.4",
```

Comment:
Probably in a different PR. We should unpin sqlalchemy. It does not use psqlgraph which is what has issues with sqlalchemy

Reply:
I'll add a TODO.